azure/template.json

```
@@ -0,0 +1,215 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "resourceEnvironmentName": {
+      "type": "string"
+    },
+    "serviceName": {
+      "type": "string"
+    },
+    "environmentName": {
+      "type": "string"
+    },
+    "sharedAppServicePlanName": {
+      "type": "string"
+    },
+    "sharedResourceGroup": {
+      "type": "string"
+    },
+    "configurationStorageConnectionString": {
+      "type": "securestring"
+    },
+    "certificateThumbprintsToLoad": {
+      "type": "string"
+    },
+    "loggingRedisConnectionString": {
+      "type": "securestring"
+    },
+    "loggingRedisKey": {
+      "type": "string"
+    }, 
+    "storageConnectionString": {
+      "type": "string"
+    },
+    "tokenCertificateThumbprint": {
+      "type": "string"
+    }
+  },
+  "variables": {
+    "deploymentUrlBase": https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
```

You've lost a double quote here
```suggestion
    "deploymentUrlBase": "https://raw.githubusercontent.com/SkillsFundingAgency/das-platform-building-blocks/master/templates/",
```

---

Pulled in your recommended fix 👍